### PR TITLE
MODCONSKC-25. Sync mod-consortia merged PRs with mod-consortia-keycloak

### PR DESCRIPTION
**Pull request by SerhiiNosko**
## Purpose
https://folio-org.atlassian.net/browse/MODCONSKC-25

## Approach
There is some PRs that was merged in mod-consortia and not merged to mod-consortia keycloak.
Need to analyze all recent PRs and move changes to mod-consortia-keycloak.

For future:
Need to setup a rule for authors of PRs in mod-consortia to create PR clone in mod-consortia-keycloak and merge 2 PRs together

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -568,6 +569,12 @@
       ],
       "visible": true
     },
+    {
+      "permissionName": "consortia.inventory.update.ownership",
+      "displayName": "Inventory: Update ownership",
+      "description": "Inventory: Update ownership",
+      "visible": true
+    },
```
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:577]: Related PR: https://github.com/folio-org/mod-consortia/pull/155

File: src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java [hunk 2]
```diff
@@ -86,6 +86,7 @@ public SharingInstance start(UUID consortiumId, SharingInstance sharingInstance)
         String source = switch (inventoryInstance.get("source").asText().toLowerCase()) {
           case "folio" -> SourceValues.CONSORTIUM_FOLIO_INSTANCE.getValue();
           case "marc" -> SourceValues.CONSORTIUM_MARC_INSTANCE.getValue();
+          case "linked_data" -> SourceValues.CONSORTIUM_LINKED_DATA_INSTANCE.getValue();
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java:89]: Related PR: https://github.com/folio-org/mod-consortia/pull/154

File: src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java [hunk 3]
```diff
@@ -56,8 +56,7 @@ public class TenantServiceImpl implements TenantService {
   private static final String TENANTS_IDS_NOT_MATCHED_ERROR_MSG = "Request body tenantId and path param tenantId should be identical";
 
   private static final String DUMMY_USERNAME = "dummy_user";
-  @Value("${folio.system-user.username}")
-  private String systemUserUsername;
+  private static final String SYSTEM_USERNAME = "mod-consortia-keycloak";
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java:59]: In Eureka system user is created automatically with username that equals to module name. So it is not necessary now to populate correct system username in application.yml

File: descriptors/ModuleDescriptor-template.json [hunk 4]
```diff
@@ -668,6 +679,9 @@
         "inventory-storage.holdings-types.item.post",
         "inventory-storage.holdings-types.item.put",
         "inventory-storage.holdings-types.item.delete",
+        "inventory-storage.holdings-sources.item.post",
+        "inventory-storage.holdings-sources.item.put",
+        "inventory-storage.holdings-sources.item.delete",
```
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:684]: All these 7 permissions were in file system-user-permissions but were missed in ModuleSecriptor-template

**Review by dmtkachenko — APPROVED** [review]

**Review by BKadirkhodjaev — APPROVED** [review]

**Pull request merged**